src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -35,76 +37,78 @@
           />
           <div v-else-if="isEmpty(deliveryAddress)" class="info-text">(Not entered)</div>
           <div v-else class="info-text">Same as Mailing Address</div>
-        </v-flex>
+        </v-col>
+
+        <template v-if="!isSummaryView">
```

Comment:
I see a flag `isEditing` and this flag `isSummaryView`. Are they mutually exclusive? Do they interact (what are the possible values)?

It might be easier to understand with a sort of state machine like I did for Nature of Business. Then you know exactly which condition (state) the code applies to.

Reply:
They are their own states. They don't interact directly no. They are booleans. 
One is an indicator if the component is in its form state, the other to indicate a summary state. 

I'm not sure i want to add complexity, as they are both rather simple flags. I can have a look at what you are referring too though!

Reply:
Thanks for explanation. I didn't realize you could have a summary view while editing.

If you come up with a way to comment or self-document this to reduce future confusion, great. Otherwise, OK for now I guess.

Reply:
ok for sure, will have a look. I think a comment where defined would help then!

Reply:
Updated prop comment, although the comment on `isEditing` seems pretty good.
To 'summarize' (no pun intended) `isSummaryView` is a 3rd state, where the component is in its static form but WITHOUT its actions to initialize the editing mode. 

This is an alternative to an additional summary component to display the information obviously.